squin_op_playground.py

```diff
@@ -0,0 +1,59 @@
+from kirin import ir, types
```

Comment:
you wanna delete the playground?

Reply:
Yes! Thank you for catching that. I should probably do a local ignore for an `_playground.py` files moving forward

Reply:
I usually use a `main.py` as my playground it's in the `.gitignore` already IIRC